src/testsuite/vec_bcd_dummy.c

```diff
@@ -37,3 +73,68 @@ test_vec_bcddiv (vui32_t a, vui32_t b)
 {
   return vec_bcddiv (a, b);
 }
+
+#if (__GNUC__ > 4)
+_Decimal128
+test__builtin_ddedpdq ( _Decimal128 value)
+{
+  return __builtin_ddedpdq (2, value);
+}
+
+_Decimal128
+test__builtin_denbcdq ( _Decimal128 value)
+{
+  return __builtin_denbcdq (1, value);
+}
+
+#ifdef _ARCH_PWR8
+vi128_t
+test__builtin_bcdadd (vi128_t vra, vi128_t vrb)
+{
+  return __builtin_bcdadd (vra, vrb, 0);
+}
+
+int
+test__builtin_bcdadd_eq (vi128_t vra, vi128_t vrb)
+{
+  return __builtin_bcdadd_eq (vra, vrb, 0);
+}
+
+vi128_t
+test__builtin_bcdsub (vi128_t vra, vi128_t vrb)
+{
+  return __builtin_bcdsub (vra, vrb, 0);
+}
+
+int
+test__builtin_bcdsub_eq (vi128_t vra, vi128_t vrb)
+{
+  return __builtin_bcdsub_eq (vra, vrb, 0);
+}
+
+vi128_t
+test__builtin_bcdmax (vi128_t vra, vi128_t vrb)
+{
+  vi128_t result = vra;
+  if (__builtin_bcdsub_lt (vra, vrb, 0))
+    result = vrb;
+
+  return result;
+}
+
+vi128_t
+test__builtin_bcdabsd (vi128_t vra, vi128_t vrb)
+  {
+    //vi128_t vra_mb,  vrb_ma;
+    vi128_t result;
+    //vra_mb = __builtin_bcdsub (vra, vrb, 0);
+    //vrb_ma = __builtin_bcdsub (vrb, vra, 0);
```

Comment:
any reason to keep the commented-out code?

Reply:
no, just forgot to go back and clean up.
Fixed